app.py

```diff
@@ -335,6 +345,21 @@ def upload_files_proc(upload_files):
     geometry_aoi_list = []
 
     for upload_file in upload_files:
+        file_name = getattr(upload_file, 'name').lower()
+        upload_file.seek(offset=0)
+
+        # Parse GPKG file
+        if file_name.endswith('.gpkg'):
+            with tempfile.NamedTemporaryFile(suffix='.gpkg') as tmp:
+                tmp.write(upload_file.getbuffer())
+                geometry = parse_gpkg_coords_to_ee_geometry(file=file_name)
```

Comment:
this attempts at opening the `.gpkg `file but it ends up calling the name 'file.gpkg' instead of calling the actual temporary file through its path again, get the name of the temp file instead of the upload, use `tmp.name` to parse through it using your parsing function